docs/docs/08.1-more-about-refs.md

```diff
@@ -31,7 +31,7 @@ myComponentInstance.doSomething();
 
 > Note:
 >
-> This should only ever be used at the top level. Inside components, let your `props` and `state` handle communication with child components, or use one of the other methods of getting a ref (string attribute or callbacks).
+> This should only ever be used at the top level. Inside components, let your `props` handle communication with child components, or use one of the other methods of getting a ref (string attribute or callbacks).
```

Comment:
Well, you can use your own state to communicate to the children, which is what that's trying to say. You would only do this via the children's props. I see that this isn't clear but I don't think your edit is technically right either.